src/App.tsx

```diff
@@ -0,0 +1,144 @@
+import "./App.css";
+import { CardFilm } from "./Components/molecules/cardFilm";
+import { Filter } from "./Components/molecules/filter";
+import { Header } from "./Components/molecules/header";
+import { ShortCardFilm } from "./Components/molecules/shortCardFilm";
+import { Sidebar } from "./Components/molecules/sidebar";
+import { StarRating } from "./Components/molecules/starRating";
+import { TrailerFilm } from "./Components/molecules/trailerFilm";
+import { films, trailer } from "./mock";
+import { useState } from "react";
+import { Button } from "./Components/atoms/button/Button";
+import { filterBy, sortBy } from "./helper/helper";
+import { FilterFilm } from "./Components/atoms/filterFilmBtn/FilterFilm";
+
+function App() {
+  const filterProps = {
+    countries: Array.from(
```

Comment:
Array.from(
      new Set(films.map((film) => film.country.split(", ")).flat())
    )
    
    please take it to the helper